### DISCUSSION TITLE
YANG path browser

### DISCUSSION THREAD

**Discussion by walidamer711** [karimra/gnmic#629] [Q&A] [answered]
I am getting the following error when I try to browser the YANG model supported 
gnmic path -m nokia-state --file nokia-state-combined.yang | head -10

Error: unknown shorthand flag: 'm' in -m

If I try it without -m option I get this error:
gnmic path  --file nokia-state-combined.yang | head -10
Error: stat /root/nokia-state-combined.yang: no such file or directory

**Comment by karimra** [comment]
The `-m` flag is not supported anymore, I will remove it from the examples, sorry about that.

The `--file` flag expects a path to the yang file, either pass a valid path to the flag or make sure the file is in your $PWD.

**Comment by walidamer711** [comment]
I downloaded the file and tried again but getting another error:

gnmic path --file nokia-state-combined.yang | head -10
yang processing error: no such module: ietf-inet-types
Error: yang processing failed with 1 errors

gnmic version
version : 0.25.1
 commit : 188b023
   date : 2022-06-14T02:22:58Z
 gitURL : https://github.com/karimra/gnmic
   docs : https://gnmic.kmrd.dev

**Comment by karimra** [comment, marked as answer]
try this:

```bash
git clone https://github.com/nokia/7x50_YangModels
cd 7x50_YangModels/latest_sros_22.5/

gnmic path --file nokia-combined/nokia-state.yang --dir ietf/ | head -10
```

**Comment by walidamer711** [comment]
That works. Thanks Karim!
